src/background.tex

```diff
@@ -66,7 +66,7 @@ \section{Expanding the Kubernetes API}
 
 The first way of expanding the Kubernetes API, introduced in Kubernetes 1.7, is using the
 aggregation layer. Thanks to this tool, one can perform API aggregation at runtime. The aggregation
-layer enables the creation of custom resources by deploying its own API server.
+layer enables the creation of Custom Resources by deploying its own API server.
```

Comment:
Either `CustomResources` or `custom resources`, everywhere.

Reply:
`custom resources`, definitely not `CustomResources` - there's no k8s object with that name.